locales/en/kafka.json

```diff
@@ -17,12 +18,14 @@
     "consumer_id": "Consumer ID",
     "consumergroup_successfully_deleted": "Successfully deleted consumer group {{name}}",
     "current_offset": "Current offset",
+    "dead_state": "Dead",
```

Comment:
For these states maybe group them in a states key? Perhaps at the root level of the kafka keys, I feel like state is something we might end up showing in other places not just into the drawer

Reply:
these strings are not related to this PR, maybe due to formatting strings being shuffled, can I group them in another PR?
Also, there are some strings related to the service account which is defined in kafka.json, can I move those to another PR

Reply:
Oh ok sure